servers/rendering/rendering_device.h

```diff
@@ -393,11 +393,14 @@ class RenderingDevice : public Object {
 	/*****************/
 
 	enum BarrierMask {
-		BARRIER_MASK_RASTER = 1,
-		BARRIER_MASK_COMPUTE = 2,
-		BARRIER_MASK_TRANSFER = 4,
+		BARRIER_MASK_VERTEX = 1,
+		BARRIER_MASK_FRAGMENT = 2,
+		BARRIER_MASK_COMPUTE = 4,
+		BARRIER_MASK_TRANSFER = 8,
```

Comment:
These enums are exposed to scripting

https://github.com/godotengine/godot/blob/b7032b5ecf8ac247e01acd5e54c9028c0dfe52df/servers/rendering/rendering_device.cpp#L1074

So the bindings need to be updated as well

Reply:
This breaks compat then. How do we want to handle it?

Reply:
I'm not sure, are bitfield flags and integers interchangeable? If not, then its not really breaking compatibility. All the old bitfields are still there, they have just been renumbered internally.

Reply:
Oops, easy fix :)